nx-X11/programs/Xserver/hw/nxagent/Rootless.c

```diff
@@ -351,14 +351,14 @@ void nxagentRootlessRestack(unsigned long children[], unsigned int nchildren)
 
   for (i = 0; i < ntoplevel; i++)
   {
-    fprintf(stderr, "[%p]\n", toplevel[i]);
+    fprintf(stderr, "nxagentRootlessRestack: [%p]\n", toplevel[i]);
```

Comment:
Well, there are about 100000 locations all over the place where this could be done. But not in this commit.